src/test/java/org/jabref/gui/autocompleter/SuggestionProvidersTest.java

```diff
@@ -0,0 +1,49 @@
+package org.jabref.gui.autocompleter;
+
+import org.jabref.logic.journals.JournalAbbreviationRepository;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.FieldFactory;
+import org.jabref.model.entry.field.SpecialField;
+import org.jabref.model.entry.field.StandardField;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.mockito.Mockito.mock;
+
+class SuggestionProvidersTest {
+
+  @Test
+  void getForFieldTest() {
+      BibDatabase database = new BibDatabase();
+      JournalAbbreviationRepository abbreviationRepository = mock(JournalAbbreviationRepository.class);
+      BibEntry entry = new BibEntry();
+      Field personEntryField = StandardField.AUTHOR;
+      Field singleEntryField = StandardField.XREF;
+      Field multipleEntryField = StandardField.XDATA;
+      Field journalEntryField = StandardField.JOURNAL;
+      Field publisherEntryField = StandardField.PUBLISHER;
+      Field specialEntryField = SpecialField.PRINTED;
```

Comment:
Wrapping the StandardFields in another Fields var seems way too 'over-abstracted' to me... StandardField.AUTHOR etc. is imho already a sufficient level of abstraction and I don't see a need to change the personEntryField to another StandardField but AUTHOR, since the test seems comprehensive enough to me.